### PR TITLE
Fix the 'cut-plane compute folder' command and create test output files in the proper directories

### PR DESCRIPTION
**Pull request by adrien-berchet**

### CODE REVIEW

**Comment by adrien-berchet** [comment]
I can't reproduce the CI error locally :thinking:

**Comment by mgeplf** [comment]
> I can't reproduce the CI error locally

It probably will fail on the bb5; iirc, you need a numpy that has this: https://github.com/numpy/numpy/pull/23707/files#diff-81215d563ecc5ed68da270adc4bc3202f61a0aceacbf235c3ff6c314381f838eR43

**Comment by adrien-berchet** [comment]
> > I can't reproduce the CI error locally
> 
> It probably will fail on the bb5; iirc, you need a numpy that has this: https://github.com/numpy/numpy/pull/23707/files#diff-81215d563ecc5ed68da270adc4bc3202f61a0aceacbf235c3ff6c314381f838eR43

Ah, this again... :sweat: 
I'll see later how to handle that

**Comment by adrien-berchet** [comment]
It's not clean but I found a way :)

File: tests/cut-plane/test_detection.py [hunk 1]
```diff
@@ -80,9 +80,17 @@ def test_minimize_more17():
     params = 4, 45, -21, 0, 0, 61
 
     result = _minimize(params, _get_points(), bin_width=10)
-    assert_allclose(result,
-                    [4.1510e+00, 4.6763e+01, -2.0966e+01, -2.7207e-04, -4.5070e-04, 6.8804e+01],
-                    rtol=1e-4)
+    if (
+        version.parse(np.__version__) < version.parse('1.25')
+        or "AVX512_SKX" not in np.show_config(mode="dicts").get("SIMD Extensions", {}).get("found", {})
+    ):
```
Comment (adrien-berchet) [inline tests/cut-plane/test_detection.py:86]: @mgeplf is this ok for you?

Comment (mgeplf) [reply tests/cut-plane/test_detection.py:86]: > It's not clean but I found a way :)

Cool and scary at the same time :)

> @mgeplf is this ok for you?

Yeah, let's get this unblocked.  If we have a chance to find a better method, that would be great.

**Review by mgeplf — APPROVED** [review]

**Comment by adrien-berchet** [comment]
Thanks!

**Comment by mgeplf** [comment]
No, thank you!

**Pull request merged**